src/components/__tests__/FormRegisterCoordinator.cy.js

```diff
@@ -0,0 +1,885 @@
+        .and('have.color', grey10)
+        .and(
+          'contain',
+          i18n.global.t('register.coordinator.form.labelResponsibility'),
+        );
+      // checkbox border
+      cy.dataCy('form-register-coordinator-responsibility')
+        .find('.q-checkbox__bg')
+        .should('have.css', 'border-radius', '4px');
+      // checkbox height
+      cy.dataCy('form-register-coordinator-responsibility')
+        .find('.q-checkbox__bg')
+        .invoke('height')
+        .should('be.equal', 18);
+      // checkbox width
+      cy.dataCy('form-register-coordinator-responsibility')
+        .find('.q-checkbox__bg')
+        .invoke('width')
+        .should('be.equal', 18);
+    });
+
+    it('renders checkbox terms', () => {
+      cy.dataCy('form-register-coordinator-terms')
+        .should('be.visible')
+        .find('.q-checkbox__label')
+        .should('be.visible')
+        .and('have.color', grey10)
+        .and('contain', i18n.global.t('register.coordinator.form.labelTerms'));
+      // checkbox border
+      cy.dataCy('form-register-coordinator-terms')
+        .find('.q-checkbox__bg')
+        .should('have.css', 'border-radius', '4px');
+      // checkbox height
+      cy.dataCy('form-register-coordinator-terms')
+        .find('.q-checkbox__bg')
+        .invoke('height')
+        .should('be.equal', 18);
+      // checkbox width
+      cy.dataCy('form-register-coordinator-terms')
+        .find('.q-checkbox__bg')
+        .invoke('width')
+        .should('be.equal', 18);
+    });
+
+    it('allows user to reveal and hide password', () => {
+      testPasswordInputReveal({
+        identifierPassword: 'form-register-coordinator-password',
+      });
+    });
+
+    it('allows user to reveal and hide password confirm', () => {
+      testPasswordInputReveal({
+        identifierPassword: 'form-register-coordinator-password-confirm',
+      });
+    });
+
+    it('validates company correctly', () => {
+      // fill in other parts of the form to be able to test company
+      cy.dataCy('form-register-coordinator-first-name-input').type('John');
+      cy.dataCy('form-register-coordinator-last-name-input').type('Doe');
+      // empty company
+      cy.dataCy('form-register-coordinator-submit')
+        .should('be.visible')
+        .click();
+      cy.dataCy('form-register-coordinator-company')
+        .find('.q-field__messages')
+        .should('be.visible')
+        .and(
+          'contain',
+          i18n.global.t('register.coordinator.form.messageFieldRequired', {
+            fieldName: i18n.global.t(
+              'register.coordinator.form.labelCompanyShort',
+            ),
+          }),
+        );
+      cy.dataCy('form-register-coordinator-company').click();
+      cy.get('.q-menu .q-item').first().click();
+      cy.get(
+        '*[data-cy="form-register-coordinator-company"] .q-field__messages [role="alert"]',
+      ).should('not.exist');
+    });
+
+    it('validates email correctly', () => {
```

Comment:
We don't need this extensively duplicated email tests here. Maybe we can create simple standalone component e.g emailFormField.vue with corresponding component tests to avoid duplicate tests.

Reply:
Refactored email and phone inputs into standalone components.
These will be used in other forms too.